apps/gitness/src/hooks/useAPIPath.ts

```diff
@@ -1,6 +1,15 @@
 import { useMFEContext } from '../framework/hooks/useMFEContext'
 
 export function useAPIPath(path: string) {
+  const mfeContext = useMFEContext()
+  const isMFE = mfeContext.renderUrl !== ''
+  if (isMFE) {
+    return `/code${path}`
+  }
+  return path
+}
+
+export function useExtendedAPIPath(path: string) {
```

Comment:
why do we need two separate functions doing such similar things? let's discuss before we merge this.